src/AzSK.ADO/Framework/Core/SVT/SVTResourceResolver.ps1

```diff
@@ -229,10 +264,22 @@ class SVTResourceResolver: AzSKRoot {
                 }
                 $TotalSvc = 0;
                 $ScannableSvc = 0;
+                # fetching the installed extensions count for the organization
+                $resourceURL = "https://extmgmt.dev.azure.com/$($this.organizationName)/_apis/extensionmanagement/installedextensions?api-version=6.0-preview.1"
+                try { 
+                    $responseList = [WebRequestHelper]::InvokeGetWebRequest($resourceURL) ;
+                }
+                catch {}
+                # storing the data into a data structure (orgTelemetryData)
+                $this.orgTelemetryData["installed_extensions"] = $responseList.Length
+                $this.orgTelemetryData["organization_name"] = $this.organizationName
                 foreach ($thisProj in $projects) 
                 {
                     $projectName = $thisProj.name
                     $projectId = $thisProj.id;
+                    # getting all the resources count
+                    # and sending them to telemetry as well
+                    $this.GetResourceCount($projectName)
```

Comment:
This should be called at the end after all the resources have been loaded into memory. Also this method should be called only if perf telemetry is enabled.